README.md

```diff
@@ -576,4 +576,4 @@ withFx(app)(state, actions).foo()
 
 ## License
 
-Hyperapp Effects is MIT licensed. See [LICENSE](LICENSE.md).
\ No newline at end of file
+Hyperapp Effects is MIT licensed. See [LICENSE](LICENSE.md).
```

Comment:
What's the change here? Trailing newline?

Reply:
perhaps so.

i didnot edit this line

Reply:
Looks like it, not a problem.

Reply:
Prettier?